test/Hyperfund.t.sol

```diff
@@ -29,9 +30,16 @@ contract HyperfundTest is Test {
 
     function setUp() public {
         manager = address(this);
-        hypercertMinter = new HypercertMinter();
-        baseHypercertId =
-            hypercertMinter.mintClaim(address(this), totalUnits, "uri", IHypercertToken.TransferRestrictions.AllowAll);
+        vm.recordLogs();
+
+        // hypercertminter address in Sepolia
+        hypercertMinter = IHypercertToken(0xa16DFb32Eb140a6f3F2AC68f41dAd8c7e83C4941);
+        assertEq(keccak256(abi.encodePacked(hypercertMinter.name())), keccak256("HypercertMinter"));
+
+        hypercertMinter.mintClaim(manager, totalUnits, "uri", IHypercertToken.TransferRestrictions.AllowAll);
```

Comment:
why did you change this to manager?

Reply:
it was address(this), and manager is address(this) so its same.

Reply:
oh actually that line needs to be removed, manager is defined before